app/js/showControllers.js

```diff
@@ -46,6 +46,10 @@ angular.module('modulusOne.showControllers', [
       })
     }
 
+    $scope.cancelEdit = function(toggleEditWhenDone) {
+      window.location.reload();
```

Comment:
Yeah, try the `$state.reload()` method of the view router we use:
https://angular-ui.github.io/ui-router/site/#/api/ui.router.state.$state